### DISCUSSION TITLE
security features and update frequency

### DISCUSSION THREAD

**Discussion by morpheuslord** [Floorp-Projects/Floorp#688] [Q&A]
I'm aware of several security features like tracker blockers, but I'm curious about the frequency of updates. My main concern is relying on outdated listings of trackers or other information that doesn't receive regular updates. I'm enthusiastic about the project and strongly encourage others to use it because it's really impressive.

**Comment by Ryosuke-Asano** [comment]
Ahhh... So Need more privacy protection?

**Reply by morpheuslord** [reply]
yes, like if there are any already present they need constant updates. I checked the adblockers and out of 150, the browser could block 50 of them. It needs to have updated listings to actively protect against or block.

**Reply by Ryosuke-Asano** [reply]
Ahh. Floorp doesn't Block ads.

**Reply by creeper-0910** [reply]
Previously, uBlock origin was installed as standard, but in recent versions it has become optional.

**Reply by morpheuslord** [reply]
Well, I mentioned Adblock as an example. Having an adblock is dope, but filtering online traffic to avoid malicious adware and possible malicious links is the end goal. Including these features is what makes a good browser the best browser (well, for me at least). I keep looking into stupid stuff as a hacker, and such features are a must. Combined with the aesthetic options and customization options, this would be my go-to browser.

**Reply by ioc2e3** [reply]
> 以前，uBlock origin 是作為標準安裝的，但在最近的版本中它已成為可選的。

Excuse me ...

Why is this ?

.

**Comment by ioc2e3** [comment]
> ... 。我主要擔心的是依賴過時的追蹤器清單或其他不定期更新的資訊。 ...

Also worried about these two issues ...

.
